src/how-to-build-and-run.md

```diff
@@ -36,9 +36,10 @@ debug-assertions = true
 # performance perhaps (less inlining) but it's worth it.
 codegen-units = 0
 
-# I always enable full debuginfo, however the line debuginfo (which is a more important part
-# and enables e.g. line numbers in backtraces) is enabled by `debuginfo-level = 1` too.
-debuginfo-level = 2
+# This enables full debuginfo (`debuginfo-level = 2`). The line debuginfo (which is a
+# more important part and enables e.g. line numbers in backtraces) is also enabled by
+# `debuginfo-level = 1`.
+debug = true
```

Comment:
Note that `debuginfo-level = 2` slows down the build, and `debug = true` slows down the build even more because it enables debug assertions, while `debuginfo-level = 1` is pretty much free.

Reply:
Hmm... that's good to know. I updated the PR. Thanks :)

Reply:
:+1: to me now, @petrochenkov ?